translations/translation_utility.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+# You may add your name if you have contributed to this file.
+echo -ne "
+    ╔══════════════════════════════════════╗
+    ║                                      ║
+    ║  Linux/macOS Translation utility 1.0 ║
+    ║     Powered by David Copperfield     ║
+    ║           A MuseScore User           ║
+    ║                                      ║
+    ╚══════════════════════════════════════╝\n\n"
+# Functions
+function Countdown() { # Countdown timer, format: Countdown [time in sec]
+    _countdown=$1
+    while [[ $_countdown -gt 0 ]]; do
+        echo -ne "The programme will exit in $_countdown sec."
+        ((_countdown--))
+        sleep 1 # time interval
+        echo -ne "\r"
+    done
+}
+function Confirmation() { # Process Yes/No/Quit response
+    while :; do
+        echo -e "Confirm? (y/n) "
+        read -p "   "
+        case ${REPLY} in
+        [Yy]* | [Oo][Kk] | 1) # Approved
+            return 3
+            ;;
+        [Qq]uit* | [Ee]sc* | [Ee]xit | -1) # Leave
+            Countdown 5
+            exit
+            ;;
+        "") # Avoid mis-press the Enter
+            continue
+            ;;
+        *) # Not confirmed
+            return
+            ;;
+        esac
+    done
+}
+# Functions End
+
+# ===== Modules ====
+Check_bin() { # Check if the bin path exists
+    bin_exists=$(which lupdate)
+    if [[ $bin_exists ]]; then
+        bin_path=$(dirname "$bin_exists")
+    else
+        echo -e "\033[35mlupdate / lrelease not in PATH\033[0m"
```

Comment:
Drop this line, it'd be confusing if the next step, esp. for Mac, is successful, and if not another error merssage shows anyhow

Reply:
Do you mean the colours?￼

Reply:
No, the entire line